defaults/main.yml

```
@@ -0,0 +1,36 @@
+---
+synapse_user: synapse
+synapse_install_dir: /opt/synapse
+synapse_data_dir: /etc/synapse
+synapse_default_log_config:
+  version: 1
+  formatters:
+    precise:
+      format: '%(asctime)s - %(name)s - %(lineno)d - %(levelname)s - %(request)s - %(message)s'
+  filters:
+    context:
+      (): synapse.util.logcontext.LoggingContextFilter
+      request: ""
+  handlers:
+    file:
+      class: logging.handlers.RotatingFileHandler
+      formatter: precise
+      filename: "{{ synapse_data_dir }}/homeserver.log"
```

Please place logs in `/var/log/synapse`, not in `/etc` or user-configurable
```suggestion
      filename: "/var/log/synapse/homeserver.log"
```